package.json

```diff
@@ -91,6 +91,7 @@
     "sqlite3": "5.0.2",
     "stylelint": "^14.0.1",
     "stylelint-config-standard": "^23.0.0",
+    "truffle": "^5.4.22",
```

Comment:
What's the purpose of adding Truffle? We're using Hardhat, Truffle is a competing development framework